store/postgres/migrations/2022-03-09-07000_call_cache_block_number_idx/up.sql

```diff
@@ -0,0 +1,18 @@
+create index if not exists
+	eth_call_cache_block_number_idx
+on
+	eth_call_cache(block_number);
+
+do $$
+declare
+	tables cursor for select namespace from ethereum_networks;
```

Comment:
One more thing I just noticed: for chains that are stored in the shared/public tables, this will also try to create the migration, and fail. The query needs to be `select namespace from ethereum_networks where namespace != 'public'`